src/interactive/AccordionPanel.jsx

```diff
@@ -88,6 +81,25 @@ class AccordionPanel extends React.Component {
 		this.forceUpdate();
 	}
 
+	/**
+	 * @description set height state when the panel receives a change
+	 * @returns {undefined}
+	 */
+	componentWillReceiveProps(nextProps) {
+		if (nextProps.isOpen !== this.props.isOpen) {
+			this.setState(()=>
+				this.getPanelStyle(!nextProps.isOpen, this.contentEl),
+				() => {
```

Comment:
this extra anonymous function may not be necessary; I believe you should be able to directly call the `setTimeout` in the `setState` callback

Reply:
I removed it and it broke. Keeping it in